templates/bot/js/notification-function-base/README.md

```diff
@@ -46,238 +40,78 @@ This section walks through the generated code. The project folder contains the f
 | `bot` | The source code for the notification Teams application |
 | `templates` | Templates for the Teams application manifest and for provisioning Azure resources |
 
-The notification implementation is in `bot` folder.
-
-The following files provide the business logic for notifications. These files can be updated to fit your business logic requirements. The default implementation provides a starting point to help you get started.
+The following files can be customized and demonstrate an example implementation to get you started.
 
 | File | Contents |
 | - | - |
-| `*Trigger/function.json` | Azure Function bindings for the notification trigger |
-| `src/*Trigger.js` | Notification trigger implementation |
+| `src/index.js` | Application entry point and `restify` handlers for notifications |
```

Comment:
Azfunc-based notification bot does not have `restify` handlers.

Reply:
Oh yes, messed up when copy paste, I'll update

Reply:
Updated